src/lib.rs

```diff
@@ -5,19 +5,25 @@ pub use input::*;
 
 use std::io::{self, Read, Write};
 
-use ansi_term::Color;
 use ansi_term::Color::Fixed;
+use ansi_term::{Color, Style};
+
+use once_cell::sync::Lazy;
 
 use crate::squeezer::{SqueezeAction, Squeezer};
 
 const BUFFER_SIZE: usize = 256;
 
-const COLOR_NULL: Color = Fixed(242); // grey
-const COLOR_OFFSET: Color = Fixed(242); // grey
-const COLOR_ASCII_PRINTABLE: Color = Color::Cyan;
-const COLOR_ASCII_WHITESPACE: Color = Color::Green;
-const COLOR_ASCII_OTHER: Color = Color::Purple;
-const COLOR_NONASCII: Color = Color::Yellow;
+static STYLE_NULL_16: Lazy<Style> = Lazy::new(|| Color::Black.bold()); // grey
+static STYLE_OFFSET_16: Lazy<Style> = Lazy::new(|| Color::Black.bold()); // grey
+
+static STYLE_NULL_8BIT: Lazy<Style> = Lazy::new(|| Fixed(242).normal()); // grey
+static STYLE_OFFSET_8BIT: Lazy<Style> = Lazy::new(|| Fixed(242).normal()); // grey
+
+static STYLE_ASCII_PRINTABLE: Lazy<Style> = Lazy::new(|| Color::Cyan.normal());
+static STYLE_ASCII_WHITESPACE: Lazy<Style> = Lazy::new(|| Color::Green.normal());
+static STYLE_ASCII_OTHER: Lazy<Style> = Lazy::new(|| Color::Purple.normal());
+static STYLE_NONASCII: Lazy<Style> = Lazy::new(|| Color::Yellow.normal());
```

Comment:
At some point we should probably refactor this into a single data structure behind one `Lazy<…>` construct.

Does this change (`const` => `Lazy`) introduce any performance penalty?